public/app/core/components/ScrollBar/GrafanaScrollbar.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import GrafanaScrollbar from './GrafanaScrollbar';
+
+describe('GrafanaScrollbar', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <GrafanaScrollbar>
+          <p>Scrollable content</p>
+        </GrafanaScrollbar>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
We've been using `enzyme` in the redux-branches, `react-test-renderer` seems to do the same thing? Should we choose one of them instead of using both?